### PR TITLE
enable RPI4 support for 22.1

### PR DESCRIPTION
**Pull request by yrzr**
RPI3 and RPI4 are combined together as in FreeBSD 13 builds.

Screen Shot 2022-02-10 at 2 45 29 PM

### CODE REVIEW

File: device/RPI.conf [hunk 1]
```diff
@@ -1,34 +1,34 @@
 # https://www.raspberrypi.org/products/raspberry-pi-3-model-b/
 # https://www.raspberrypi.org/products/raspberry-pi-3-model-b-plus/
+# https://www.raspberrypi.com/products/raspberry-pi-4-model-b/
 
 export MAKE_ARGS_DEV="
```
Comment (fichtner) [inline device/RPI.conf:5]:
```suggestion
export MAKE_ARGS_DEV="
CROSS_BINUTILS_PREFIX=/usr/local/aarch64-unknown-freebsd${SRCREVISION}/bin
```

put this back for now to defer to other PR?

**Comment by fichtner** [comment]
@yrzr btw did you have any trouble with UEFI after rework of https://github.com/opnsense/tools/blob/master/build/arm.sh#L105-L108 ?

File: device/RPI.conf (hunk 1 above)
Comment (yrzr) [reply device/RPI.conf:5]: Done

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by yrzr** [comment]
@fichtner Yes, there is no `/boot/msdos/efi` directory. I have to copy it myself. Don't have time to figure it out yet.

**Comment by fichtner** [comment]
@yrzr don't see a reason why it's not there at a quick glance, but no need to rush. thanks for the feedback!

**Comment by yrzr** [comment]
@fichtner There is an error.

```
+ [ -n 'arm dvd serial vga vm' -z '' ]
[: arm dvd serial vga vm: unexpected operator
+ arm_unmount
+ sync
+ umount /usr/obj/usr/tools/config/22.1/OpenSSL:aarch64/boot/msdos
+ umount /usr/obj/usr/tools/config/22.1/OpenSSL:aarch64
+ mdconfig -d -u md0
+ echo done
done
```

**Comment by fichtner** [comment]
@yrzr sorry about this...not sure how this happend, fixed via 238c90c7b36dc

**Pull request merged**